src/wasm-lib/kcl/src/unparser.rs

```diff
@@ -281,6 +280,12 @@ impl Expr {
                 result += &e.label.name;
                 result
             }
+            Expr::AscribedExpression(e) => {
```

Comment:
The lezer grammar needs ascription support. It doesn't need to be in this PR, but it frequently gets left behind. I'll make an issue for it.

Reply:
- #5499